app/src/main/java/services/authhandler/Amplify.java

```diff
@@ -0,0 +1,20 @@
+package services.authhandler;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.amplifyframework.AmplifyException;
+import com.amplifyframework.auth.cognito.AWSCognitoAuthPlugin;
+
+public class Amplify {
+    public static void configure(Context context) {
+        try {
+            com.amplifyframework.core.Amplify.addPlugin(new AWSCognitoAuthPlugin());
+            com.amplifyframework.core.Amplify.configure(context);
+
+            Log.i("MyAmplifyApp", "Initialized Amplify");
+        } catch (AmplifyException e) {
```

Comment:
In case of an error, we probably want to notify the caller rather than just logging it. For example, if amplify isn't configured properly, our whole app basically won't work. This is something the `ConfigActivity` should handle instead of silencing the error here with a Log statement.

Reply:
Acknowledged in recent push